packages/ast-helpers/stringify.js

```diff
@@ -212,7 +212,7 @@ const es2017GeneratorJSX = {
   },
   // {...props}
   JSXSpreadAttribute(node, state) {
-    state.write('...(');
+    state.write(' {...(');
```

Comment:
is it possible that sometimes it needs the curly braces and sometime it doesn't?

Reply:
This part of the conversion is specifically for attribute spreading within JSX. I can't think of an example where we would not need the curly braces for this circumstance. I did test that spreading elsewhere in JSX does not enter this block of code, where say if you had something like <div>{[...someArray, ...someOtherArray]}</div>, this is not impacted.